03-task-manager/starter/app.js

```diff
@@ -3,19 +3,21 @@ const app = express()
 const tasks = require('./routes/tasks')
 const connectDB = require('./db/connect')
 require('dotenv').config()
+const notFound = require('./middleware/not-found')
+const errorhandlerMidddleware = require('./middleware/error-handler')
```

Comment:
It's always a good idea to proofread your code for typos, too!  Making sure things are spelled correctly ensures that anyone else using or adding to your code isn't worried about calling the wrong function.

Reply:
Sure Miles. Will do that :)